src/Console/RenderConsoleCommandOverviewMessage.php

```diff
@@ -40,16 +37,16 @@ public function __invoke(): string
         ksort($commands);
 
         foreach ($commands as $group => $commandsForGroup) {
-            $lines[] = ConsoleStyle::BOLD(ConsoleStyle::BG_BLUE(' ' . ucfirst($group) . ' '));
+            $builder->formatted(ConsoleStyle::BOLD(ConsoleStyle::BG_BLUE(' ' . ucfirst($group) . ' ')));
```

Comment:
Why don't we provide a method for this style? I'm ok if it changes a bit, I just put random something together initially.